public/index.css

```diff
@@ -0,0 +1,272 @@
+html {
```

Comment:
нужно вернуть то что было раньше
```suggestion
* {
```